middle_end/flambda2/from_lambda/lambda_to_flambda.ml

```diff
@@ -653,16 +686,15 @@ let rec cps acc env ccenv (lam : L.lambda) (k : cps_continuation)
   | Llet
       ((Strict | Alias | StrictOpt), _layout, id, _duid, defining_expr, Lvar id')
     when Ident.same id id' ->
-    (* CR sspies: dropping [debug_uid]; address in subsequent PR. *)
+    (* CR sspies: Is there any use for the debugging uid in this case? *)
```

Comment:
This is a case where a phantom let should be inserted; that could carry the duid.